src/libPMacc/examples/gameOfLife2D/include/GatherSlice.hpp

```diff
@@ -20,18 +21,15 @@
  */
 
 
-#ifndef GATHERSLICE_HPP
-#define	GATHERSLICE_HPP
+#pragma once
 
-#include "types.h"
 
 
 #include <mpi.h>
 #include "mappings/simulation/GridController.hpp"
-
-//c includes
 #include "memory/boxes/PitchedBox.hpp"
 #include "dimensions/DataSpace.hpp"
+#include "types.h"                                  // DIM*
```

Comment:
Because there is no consistent order in the existing code I just used the C++ standard include order from local to global (see [here](http://stackoverflow.com/questions/2762568/c-c-include-file-order-best-practices) or [here](http://stackoverflow.com/questions/614302/c-header-order)) because it is best to find hidden include dependencies.